packages/cactus-test-geth-ledger/tsconfig.json

```diff
@@ -7,9 +7,7 @@
     "rootDir": "./src",
     "tsBuildInfoFile": "../../.build-cache/cactus-test-geth-ledger.tsbuildinfo"
   },
-  "include": [
-    "./src"
-  ],
+  "include": ["./src", "src/**/*.json"],
```

Comment:
@outSH Why exclude .json files? (I'm just curious)

Reply:
@petermetz I'm including json files to import smart contract ABI in the test

Reply:
@outSH Aargh, I somehow read the diff backwards, I thought you were making the opposite change. Sorry :-)